prow/jobs/incubator/reconciler/reconciler.yaml

```diff
@@ -3,6 +3,61 @@
 
 presubmits: # runs on PRs
   kyma-incubator/reconciler:
+    - name: pre-main-kyma-incubator-reconciler-e2e
+      annotations:
+        pipeline.platform: "gardener_gcp"
+        pipeline.trigger: "pr-submit"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "pre-main-kyma-incubator-reconciler-e2e"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-cluster-version: "true"
+        preset-gardener-gcp-kyma-integration: "true"
+        preset-kyma-cli-stable: "true"
+      run_if_changed: '^resources'
+      skip_report: false
+      decorate: true
+      cluster: untrusted-workload
+      max_concurrency: 10
+      branches:
+        - ^master$
+        - ^main$
+      extra_refs:
+        - org: kyma-project
+          repo: kyma
+          path_alias: github.com/kyma-project/kyma
+          base_ref: main
+        - org: kyma-project
+          repo: test-infra
+          path_alias: github.com/kyma-project/test-infra
+          base_ref: main
+      spec:
+        containers:
+          - image: "eu.gcr.io/kyma-project/test-infra/pr/kyma-integration:PR-4084-k8s1.18"
```

Comment:
Update image once this [PR](https://github.com/kyma-project/test-infra/pull/4084) is approved and we have a new image foe kyma-integration.